packages/zk-certificates/lib/keyManagement.ts

```diff
@@ -134,6 +134,31 @@ export function createHolderCommitment(
   ).toString();
 }
 
+/**
+ * Decompress an EdDSA public key.
+ * @param pubKeyHex - Compressed public key as hex string.
+ * @returns The decompressed public key as [Ax, Ay] strings in decimal format.
+ */
+export async function decompressEddsaPubKey(
+  pubKeyHex: string,
+): Promise<[string, string]> {
+  const babyjub = await buildBabyjub();
+  if (pubKeyHex.length !== 64) {
+    throw new Error('Invalid public key length');
+  }
+  const hexArray = pubKeyHex.match(/.{2}/gu) ?? [];
```

Comment:
What this regex is supposed to be doing? I see several problems:

1. It seems that it check if there are even amount of symbols, but if was checked in the previous condition
2. it has Unicode flag, however hex string must not contain any Unicode symbols

If you wanted to test if the string is correct hex-encoded string, then here is more correct expression: `/^[0-9A-Fa-f]{64}$/`. It will also make the previous line unnecessary

Reply:
Good catch. That is why I love your reviews.